integration_tests/models/dbt_project_evaluator_schema_tests/core.yml

```diff
@@ -17,10 +17,6 @@ models:
         tests:
           - unique
           - not_null
-      - name: on_schema_change
```

Comment:
why remove this test?

Reply:
Because it failed 🤣 

If I recall, this is because we now handle empty strings or None differently in `wrap_string_with_quotes(str)`